recipes/hendrics/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "hendrics" %}
+{% set version = "4.0rc1.post2" %}
+{% set sha256 = "16956660f9705dc10e881b60073079ae7f4db835d811f2732623816a832c09b3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.11.0
+  run:
+    - python
+    - astropy
+    - scipy >=0.18.0
+    - numpy >=1.11.0
+    - matplotlib >=1.3
+    - stingray >=0.1rc1
+
+test:
+  imports:
+    - scipy
+    - numpy
+    - matplotlib
+    - astropy
+    - hendrics
+
+about:
+  home: http://github.com/StingraySoftware/stingray
+  license: 3-clause BSD
+  license_family: BSD
+  license_file: LICENSE.rst
+  summary: 'High-ENergy Data Reduction Interface from the Command Shell'
+
```

Comment:
Another I-wouldn't-mention-if-not-for-the-important-change: Please drop the blank line.